openmdao/core/problem.py

```diff
@@ -364,10 +364,11 @@ def get_val(self, name, units=None, indices=None, get_remote=False):
             Units to convert to before return.
         indices : int or list of ints or tuple of ints or int ndarray or Iterable or None, optional
             Indices or slice to return.
-        get_remote : bool
+        get_remote : bool or None
             If True, retrieve the value even if it is on a remote process.  Note that if the
```

Comment:
Since this argument can be True, False, or None, the docstring should clearly describe what happens if you set it to each of these three cases. (Seems to be missing False right now.).